config.toml

```diff
@@ -30,8 +30,7 @@ pygmentsStyle = "emacs"
 enableGitInfo = true
 
 # Norwegian ("no") is sometimes but not currently used for testing.
-# Hindi is disabled because it's currently in development.
-disableLanguages = ["hi", "no"]
+disableLanguages = ["no"]
```

Comment:
Will this change enable Hindi on the published website? Is that the intention of this PR?

Reply:
Yes; merging this PR will launch Hindi.

Reply:
That's great news! Should we update the PR title or description to make this clear?